packages/commerce-sdk-react/src/hooks/ShopperBaskets/cache.ts

```diff
@@ -145,7 +145,7 @@ export const cacheUpdateMatrix: CacheUpdateMatrix<Client> = {
         return {
             // TODO: Convert invalidate to an update that removes the matching basket
             invalidate: [
-                ...(customerId && !basketId
+                ...(customerId && basketId
```

Comment:
This change fixes a bug where we don't invalidate the cache after a basket is created.

Reply:
Thanks for this comment! I was under the impression that the `!` here was unintentional but this comment made me check in with @kevinxh and we found that rather than invalidate when we have a basketId, we want to be updating. But there was an issue with the updater logic. I'll push a change that reverts this and updates the update logic instead